src/Grid/grid.jl

```diff
@@ -131,13 +157,19 @@ reference_facets(::Type{<:AbstractRefShape})
 @inline reference_facets(refshape::Type{<:AbstractRefShape{2}}) = reference_edges(refshape)
 @inline reference_facets(refshape::Type{<:AbstractRefShape{3}}) = reference_faces(refshape)
 
+@inline reference_faces(::AbstractCell{refshape})    where refshape = reference_faces(refshape)
+@inline reference_edges(::AbstractCell{refshape})    where refshape = reference_edges(refshape)
+@inline reference_vertices(::AbstractCell{refshape}) where refshape = reference_vertices(refshape)
```

Comment:
Use [here](https://github.com/Ferrite-FEM/Ferrite.jl/pull/889/files#diff-13ae190cb008d749905405bddca2a9ce395b303e12c781186df0d908efd05a90L154-L171)?

Reply:
You mean update the linked code lines to use this? I think the code above is fine as it is tho. The functions here are mere convenience for cases where the access to the cell type might not that straight forward. :)

Reply:
Sure, but when we have them I think it makes sense to use them internally as well? 
```diff
- function vertices(c::AbstractCell{RefShape}) where RefShape
+ function vertices(c::AbstractCell)
    ns = get_node_ids(c)
-   return map(i -> ns[i], reference_vertices(RefShape))
+   return map(i -> ns[i], reference_vertices(c))
end
```
Is kind of nicer too, and will automatically add test coverage for these methods...